celer/tests/conv_warning/isolate_conv_warn.py

```diff
@@ -0,0 +1,12 @@
+from celer.dropin_sklearn import LogisticRegression
+from celer.tests.conv_warning.dumped_data import DICT_DATA
+
+
+for check_name, check_props in DICT_DATA.items():
+    X = check_props['X']
+    y = check_props['y']
+    C = check_props['C']
```

Comment:
``C`` is constant for all estimators (view its [def](https://github.com/mathurinm/celer/blob/700c280b69902faa0d5430336aa4501d9f53f264/celer/tests/test_logreg.py#L47))

Reply:
Aren't x and y small enough to be hard coded in your script ?

Reply:
``X`` shape is ``(100, 2)`` with floating numbers. Looking at ``dumped_data.py``, I think putting ``X`` and ``y`` in the same script would be cumbersome.

Reply:
Good point, I did not see the file. You can even save them as npy if they are large.

Start simple, with a single case of failure (rather than 3 : chances are, they fail for the same reason)